package.json

```diff
@@ -65,7 +65,7 @@
     "@babel/preset-env": "7.10.0",
     "@babel/preset-flow": "7.9.0",
     "@babel/preset-react": "7.10.0",
-    "@redhat-cloud-services/frontend-components-config": "4.4.1",
+    "@redhat-cloud-services/frontend-components-config": "4.5.2",
```

Comment:
Hello @epwinchell thank you for making the changes. 
I observed something after testing these changes locally and wanted to confirm same with you.
Here is the design screenshot of "All apps and services"  when I tested this PR locally which is same as without these changes
![image (1)](https://user-images.githubusercontent.com/5928530/141995136-5afdd4e4-52d3-4b56-96aa-6c4f80d0983b.png)

This current design of "All apps and services" is different from what is attached in the card https://issues.redhat.com/browse/RHCLOUD-16946
Hence I wanted to confirm if this is expected behaviour or do we need to update any other dependencies on our end to make the design of "All apps and services" similar to one shared in the jira card?

Reply:
Hi @PreetiW If you mean the design of the contents of the dropdown, the behavior is correct. We did merge a PR last week to fix the cropping issue: https://github.com/RedHatInsights/insights-chrome/pull/1691.  The initial Masthead dropdown styling changes were merged last week here: https://github.com/RedHatInsights/insights-chrome/pull/1677

Reply:
@epwinchell I observed that the jira card contains screenshot of stage-beta environment, and the design is as per what is mentioned in the card (attached screenshot)
![image](https://user-images.githubusercontent.com/5928530/141997457-cb55b331-e572-4f8f-9c37-c40d1ccda425.png)

but checking the design of same "All apps and services" in stage, prod-beta environment resulted in showing the design as per screenshot mentioned in the [comment](https://github.com/RedHatInsights/ros-frontend/pull/70#discussion_r750281359)
Hence I am curious around why the design go it is only different in one environment i.e stage-beta, can you please provide some inputs around it? Thanks!

Reply:
@PreetiW I'm told the code was not promoted yet. It should be only visible in ci/qa/stage beta for now.

Reply:
@epwinchell ohh okay but just wanted to inform you that it is also available on stage and prod-beta based on my observations. Thanks for the information.

Reply:
cc @Hyperkid123